components/log-viewer-webui/client/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "log-viewer-webui-client",
+  "version": "0.1.0",
+  "description": "",
+  "main": "src/index.jsx",
+  "scripts": {
+    "lint:check": "npx eslint --no-eslintrc --config package.json src webpack.config.js",
+    "lint:fix": "npx eslint --fix --no-eslintrc --config package.json src webpack.config.js",
+    "build": "webpack --define-process-env-node-env production",
+    "watch": "webpack serve"
+  },
+  "author": "YScope Inc. <dev@yscope.com>",
+  "license": "Apache-2.0",
+  "type": "module",
+  "devDependencies": {
+    "@babel/core": "^7.24.7",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-transform-runtime": "^7.24.7",
```

Comment:
I don't see those getting configured in webpack.config.js. Mind sharing what are the usages of those?

Reply:
Sorry, didn't clean up package.json properly.